WildShape-Sheets-API/Services/UserService.cs

```diff
@@ -3,14 +3,17 @@
 using MongoDB.Driver;
 using System.IdentityModel.Tokens.Jwt;
 using System.Security.Claims;
+using System.Security.Cryptography;
 using System.Text;
 using WildShape_Sheets_API.Models;
 
 namespace WildShape_Sheets_API.Services {
     public class UserService {
 
         private readonly IMongoCollection<User> users;
-        //private readonly string key;
+        const int keySize = 64;
```

Comment:
Best to have system config variables all in one place. Does IConfiguration mean something to you? ChatGPT said that is the thing to use. So I think you build a configuration interface and they reference that in your code. ChatGPT had this example:

```
public class MyController : ControllerBase
{
    private readonly IConfiguration _configuration;

    public MyController(IConfiguration configuration)
    {
        _configuration = configuration;
    }

    public IActionResult Index()
    {
        string mySetting = _configuration["MySetting"];
        // Use mySetting...

        return View();
    }
}

```